src/patternfly/components/InlineEdit/examples/InlineEdit.md

```diff
@@ -0,0 +1,352 @@
+          {{/inline-edit-input}}
+        {{/table-td}}
+        {{#> table-td table-td--action="true"}}
+          {{#> inline-edit-group inline-edit-group--modifier="pf-m-action-group pf-m-confirm"}}
+            {{> inline-edit-action-save inline-edit-action-save--IsIcon="true"}}
+            {{> inline-edit-action-cancel inline-edit-action-cancel--IsIcon="true"}}
+          {{/inline-edit-group}}
+          {{#> inline-edit-group inline-edit-group--modifier="pf-m-action-group pf-m-enable"}}
+            {{#> inline-edit-action}}
+              {{#> inline-edit-action}}
+                {{#> button button--modifier="pf-m-plain" button--attribute=(concat 'id="' inline-edit--id '-edit-button" aria-label="edit" aria-labelledby="' inline-edit--id '-label ' inline-edit--id '-edit-button"')}}
+                  <i class="fas fa-pencil-alt" aria-hidden="true" />
+                {{/button}}
+              {{/inline-edit-action}}
+            {{/inline-edit-action}}
+            {{#> inline-edit-action}}
+              {{#> dropdown id=(concat table--id "-dropdown-kebab-right-aligned-1") dropdown--IsActionMenu="true" dropdown-menu--modifier="pf-m-align-right" dropdown-toggle--modifier="pf-m-plain" dropdown--HasKebabIcon="true" aria-label="Actions"}}
+              {{/dropdown}}
+            {{/inline-edit-action}}
+          {{/inline-edit-group}}
+        {{/table-td}}
+      {{/table-tr}}
+    {{/table-tbody}}
+  {{/table}}
+{{/inline-edit}}
+```
+
+## Documentation
+
+### Accessibility
+All accessibility requirements for inputs apply to elements within inline edit.
+
+| Attribute | Applied to | Outcome |
+| -- | -- | -- |
+| `id` | `[labelling element]` | Provides a reference for toggle button and/or editable content. |
+| `aria-label="descriptive content"` | `.pf-c-inline-edit__toggle > button` | Provides an accessible name for toggle button. **Required** |
+| `aria-labelledby="[labelling element] [toggle button id]"` | `.pf-c-inline-edit__toggle > button` | Provides an accessible description for toggle button. **Required** |
+
+### Usage
+| Class | Applied to | Outcome |
+| -- | -- | -- |
+| `.pf-c-inline-edit` | `<form>`, `<div>` | Initiates the inline edit component. **Required** |
+| `.pf-c-inline-edit__value` | `*` | Initiates an inline edit value. **Required** |
+| `.pf-c-inline-edit__input` | `*` | Initiates an inline edit input. **Required** |
+| `.pf-c-inline-edit__action` | `*` | Initiates an inline edit action (visible when inline edit region is active). **Required** |
+| `.pf-c-inline-edit__value-group` | `*` | Initiates an inline edit value group. |
+| `.pf-c-inline-edit__input-group` | `*` | Initiates an inline edit input group. This group can be nested (check/radio group). |
+| `.pf-c-inline-edit__action-group` | `*` | Initiates an inline edit action group that wraps action buttons within active, inline edit regions. |
```

Comment:
if you have `pf-c-inline-edit__action-group` do you need `pf-m-action-group` as a modifier? Could you explain what they're doing.
Also I think `pf-m-action-group` can be changed to just `pf-m-action` 
Also would need to add `pf-m-action-group` to documentation

Reply:
Don't need action `pf-c-inline-edit__action-group`